client/src/partials/patients/record/patient_record.html

```diff
@@ -99,14 +99,15 @@
 
       <div class="row">
 
-        <div class="col-sm-6" style="height : 250px; overflow: auto;">
+        <div class="col-sm-6">
           <!-- find document  -->
           <bh-find-document
             patient-uuid="PatientRecordCtrl.patient.uuid"
             enable-patient-details="false"
             enable-search="false"
             enable-option-bar="true"
-            display="default">
+            display="default"
+            height="250px">
```

Comment:
Hmm..  Hardcoded styles are not a great idea.  Is there anyway to make this component take up `100%` height?

This won't block the PR, as it may require CSS changes beyond this component.